build.gradle

```diff
@@ -429,6 +403,19 @@ allprojects {
       }
     }
 
+    jcabiAspects = { ->
+      apply plugin: 'io.freefair.aspectj.post-compile-weaving'
+
+      compileJava {
+        // Turn off `adviceDidNotMatch` spam warnings
+        ajc.options.compilerArgs += '-Xlint:ignore'
+      }
+
+      dependencies {
+        aspect group: 'com.jcabi', name: 'jcabi-aspects', version: jcabiAspectsVersion
```

Comment:
Once an aspect is added as dependency, it does all the weaving magic out of the box wherever _summoned_ by an annotation like `@Loggable`